examples/stand_alone/marker/marker.cc

```diff
@@ -29,7 +29,7 @@ int main(int _argc, char **_argv)
   // Create the marker message
   ignition::msgs::Marker markerMsg;
   markerMsg.set_ns("default");
-  markerMsg.set_id(0);
+  markerMsg.set_id(1);
```

Comment:
Not directly related to this PR, but just based on my experience debugging https://github.com/osrf/gazebo/issues/2766 . Given that this example is the main documentation entry point on how to use the Gazebo marker feature, perhaps we could write in a comment here that the id 0 has a special meaning (to represet **not set id**, that result in a random id being generated and a new marker created every time) and so it should not be used?

Reply:
Agree. Given the sparse comments in the original code, I was hesitant to add many comments. However, I will add a short comment regarding the id.

Reply:
I have tried to add a note regarding set_id's behaviour. I hope that it is not too verbose. In case it is verbose, please feel free to suggest a better change. Thank you.

Reply:
I think it is great! Probably I would just move it before the first set_id, so we explain to the readed why we assign id starting from 1 .